linux_os/guide/system/software/system-tools/package_abrt-addon-ccpp_removed/rule.yml

```diff
@@ -18,6 +18,9 @@ identifiers:
     cce@rhel7: 82920-0
     cce@rhel8: 82919-2
 
+references:
+    srg: SRG-OS-000095-GPOS-00049
```

Comment:
The rule is part of the OSPP profile.  I was trying to map it to something sensible for the STIG to keep the two bodies of content in sync.

From the ABRT page:
"ABRT installs its own core dump handler via abrt-ccpp.service which when started, overrides the default value of kernel’s core_pattern. This causes C/C++ crashes to be handled by abrtd and by default prevents creation of core.* files in crashed process’ current directory."

I think we should keep this rule as is so that kernel functionality is not overridden.  The core.* files should provide sufficient crash debug data.

Reply:
Part of the threat model includes disclosure of secrets. It is mandatory under FIPS that no debug capabilities exist where someone can possibly gain access to keys. Therefore everything related to coredumps has to be turned off. All helpers have to be removed.